apps/console/src/features/notification-channels/settings/settings-section.tsx

```diff
@@ -0,0 +1,364 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { GenericIcon, useMediaContext } from "@wso2is/react-components";
+import classNames from "classnames";
+import React, { FunctionComponent, MouseEvent, PropsWithChildren } from "react";
+import { Card, Grid, Header, Icon, List, Menu, Message, SemanticICONS } from "semantic-ui-react";
+
+/**
+ * Prop-types for the settings section component. See also
+ * {@link SettingsSection.defaultProps}
+ */
+interface SettingsSectionProps extends TestableComponentInterface {
+    className?: string;
+    contentPadding?: boolean;
+    description?: string;
+    connectorEnabled?: boolean;
+    header: string;
+    icon?: any;
+    onPrimaryActionClick?: (e: MouseEvent<HTMLElement> | KeyboardEvent) => void;
+    onSecondaryActionClick?: (e: MouseEvent<HTMLElement> | KeyboardEvent) => void;
+    placeholder?: string;
+    primaryAction?: any;
+    primaryActionDisabled?: boolean;
+    primaryActionIcon?: SemanticICONS;
+    secondaryAction?: any;
+    secondaryActionDisabled?: boolean;
+    secondaryActionIcon?: SemanticICONS;
+    showActionBar?: boolean;
+    topActionBar?: React.ReactNode;
```

Comment:
```suggestion
    topActionBar?: ReactNode;
```
import from "react" and use